src/components/ContentHighlights/HighlightStepper/HighlightStepperTitleInput.jsx

```diff
@@ -11,37 +11,59 @@ const HighlightStepperTitleInput = () => {
   const highlightTitle = useContextSelector(ContentHighlightsContext, v => v[0].stepperModal.highlightTitle);
   const [titleLength, setTitleLength] = useState(highlightTitle?.length || 0);
   const [isInvalid, setIsInvalid] = useState(false);
-
-  const handleChange = (e) => {
+  const [highlightValue, setHighlightValue] = useState({
+    initialized: false,
+    highlightTitle: highlightTitle || '',
+    titleStepValidationError: undefined,
+    highlightTitleLength: titleLength || 0,
+  });
+  const handleChange = ((e) => {
+    e.persist();
```

Comment:
Is this `e.persist()` necessary? It appears to work as expected without it.

Reply:
Woops, leftover code from debugging